.ci/static-checks.sh

```diff
@@ -915,41 +915,9 @@ static_check_vendor()
 		# This does not really verify the integrity of vendored code:
 		# https://github.com/golang/go/issues/27348
 		# Once that is added we need to add an extra step to verify vendored code.
-		GO111MODULE=on go mod verify
+		go mod verify
 		return
 	fi
-
-	# All vendor operations should modify this file
-	local vendor_ctl_file="Gopkg.lock"
-
-	[ -e "$vendor_ctl_file" ] || { info "No vendoring in this repository" && return; }
-
-	info "Checking vendored code is pristine"
-
-	files=$(get_pr_changed_file_details_full || true)
-
-	# Strip off status
-	files=$(echo "$files"|awk '{print $NF}')
-
-	if [ -n "$files" ]
-	then
-		# PR changed files so check if it changed any vendored files
-		vendor_files=$(echo "$files" | grep -E "\<vendor/" || true)
-
-		if [ -n "$vendor_files" ]
-		then
-			result=$(echo "$files" | egrep "\<${vendor_ctl_file}\>" || true)
-			[ -n "$result" ] || die "PR changes vendor files, but does not update ${vendor_ctl_file}"
-		fi
-	fi
-
-	info "Checking vendoring metadata"
-
-	# Get the vendoring tool
-	go get github.com/golang/dep/cmd/dep
-
-	# Check, but don't touch!
-	dep check
```

Comment:
This block removal seems unrelated to the commit. Am I missing something?

Reply:
It is, sort of. The PR is to drop `go get` in order to be able to upgrade golang version (see the depends-on tag), which also invalidates the use of `dep`. Let me split it into another commit.